data/en/isvalid.json

```diff
@@ -6,11 +6,11 @@
 	"related":["isarray", "issimplevalue","isnumeric","isboolean","isdate","cfparam"],
 	"description":"Tests whether a value meets a validation or data type rule.",
 	"params": [
-		{"name":"type","description":"The valid format for the data.","required":true,"default":"","type":"string","values":["Any","Array","Binary","boolean","creditcard","date","time","email","eurodate","float","Numeric","guid","integer","Query","range","Regex","regular_expression","ssn","social_security_number","String","Struct","telephone","URL","UUID","usdate","variablename","xml","zipcode","component"]},
+		{"name":"type","description":"The valid format for the data.","required":true,"default":"","type":"string","values":["any","array","binary","boolean","cfc (same as component)","component","class (same as component)","closure","creditcard","date","datetime","email","eurodate (not recommended)","float","function","guid","integer","image","node (same as XML)","number","numeric","phone (same as telephone)","query","range","regex","regular_expression","ssn","social_security_number","string","struct","telephone","time","URL","UUID","usdate","variablename","XML","zip (same as zipcode)","zipcode"]},
```

Comment:
We don't want to have these additional annotations in the `values` because VSCode and Sublime use the `values` to provide autocomplete suggestions.

For the new ones you added that are just for Lucee, maybe you can add something to the engine notes saying types x,y,z are only supported on lucee.